server/eventHandlers/computeStats.ts

```diff
@@ -0,0 +1,52 @@
+import { SubstrateTypes } from '@commonwealth/chain-events';
+import { sequelize } from '../database';
+import moment from 'moment';
+
+
+const getLast30DaysStats = async(chain: String, event: String, stash: String ) => {
```

Comment:
change name to computeEventStats,
Remove 30 from name and pass the number of days as argument,